tests/uat/conflict_review_driver/main.go

```
@@ -37,6 +38,52 @@ type output struct {
+	request conflictassessment.ConflictAssessmentRequest,
+) (conflictassessment.ConflictAssessmentResponse, error) {
+	legacy := verifier.ConflictAssessmentRequest{
+		RequestID: request.RequestID,
+		CaseID:    request.CaseID,
+		Version:   request.Version,
+		Question:  request.Question,
+		Positions: make([]verifier.ConflictAssessmentPosition, 0, len(request.Positions)),
+		Evidence:  make([]verifier.ConflictAssessmentEvidence, 0, len(request.Evidence)),
+	}
+	for _, position := range request.Positions {
+		legacy.Positions = append(legacy.Positions, verifier.ConflictAssessmentPosition{
```

golangci-lint (2.12.2)

[error] S1016: should convert position (type github.com/markhuangai/dense-mem/internal/conflictassessment.ConflictAssessmentPosition) to github.com/markhuangai/dense-mem/internal/verifier.ConflictAssessmentPosition instead of using struct literal

(staticcheck)